### DISCUSSION TITLE
How to write stash metadata to video file?

### DISCUSSION THREAD

**Discussion by bobsutton124** [stashapp/stash#3451] [Q&A] [answered]
Is there a plugin or script or program that can use the metadata and covers scraped by Stashdb to write it directly to the video file attributes?

**Comment by HereIam80** [comment, marked as answer]
I don't think there is anything like that.
Also it would not really be beneficial for stashDB and stash itself to have something like that as that would change the hash of the file. You would have to rescan or recalculate the hash and update the database. For StashDB the Oshashes also wouldn't match anymore.

It would be possible but I don't think this has been requested before.

Just my 2cents.

**Comment by bobsutton124** [comment]
Is there any other program that anyone knows of that can take the Stash sql file or from an api call and write attributes to the video file?

**Reply by DogmaDragon** [reply]
Don't know any, but just keep in mind that it will create a duplicate entry for every file you modify and re-scan in Stash.

**Reply by justaguyinaz** [reply]
Would probably be worth the deconfliction work, if that were available. I wish that were possible with an existing plugin, but sounds like it still isn't a thing.
